src/tiled/editableselectedarea.h

```diff
@@ -38,6 +38,8 @@ class EditableSelectedArea : public QObject
 public:
     explicit EditableSelectedArea(MapDocument *mapDocument,
                                   QObject *parent = nullptr);
+    Q_PROPERTY(QRect boundingRect READ boundingRect)
```

Comment:
Please move the `Q_PROPERTY` definition up between `Q_OBJECT` and the `public:`, where we usually put it.